File: 0389-find-the-difference/0389-find-the-difference.py
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        d={}
        for i in t :
            d[i]=d.get(i,0) +1
        for i in s :
            d[i]=d.get(i,0) -1
        for x in d :
            if d[x]==1 :
                return x
        return 0

File: 0389-find-the-difference/0389-find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        int sm=0;
        for (int i=0;i<t.length();i++)
        {
            sm += t.charAt(i);
        }
        for (int i=0;i<s.length();i++)
        {
            sm -= s.charAt(i);
        }
        return (char)sm ;
    }
}